velox/functions/lib/SubscriptUtil.h

```diff
@@ -300,6 +300,7 @@ class SubscriptImpl : public exec::VectorFunction {
     // Get map keys.
     auto mapKeys = baseMap->mapKeys();
     exec::LocalSelectivityVector allElementRows(context, mapKeys->size());
+    allElementRows->setAll();
```

Comment:
Could you add a unit test for this? Thanks!

Reply:
Hi Xiaoxuan, Thank you for the review! This bug was actually a corner case found by the Velox expression fuzzer. I tried but was not able to reproduce it in a unit test with simple expressions, because I believe this error was trigger by reusing a uninitialized selectivity vector in the pool produced by a previous query. On the other hand, we can reproduce this error via the expression fuzzer (with changes in the next diff in my stack) and I have confirmed this error went away after the fix in this PR. I've updated the test plan in the internal diff.